docs/rules/README.md

```diff
@@ -229,6 +228,7 @@ These rules are only relevant to ES6 environments.
 * [no-const-assign](no-const-assign.md) - disallow modifying variables that are declared using `const` (recommended)
 * [no-dupe-class-members](no-dupe-class-members.md) - disallow duplicate name in class members (recommended)
 * [no-new-symbol](no-new-symbol.md) - disallow use of the `new` operator with the `Symbol` object (recommended)
+* [no-restricted-imports](no-restricted-imports.md) - restrict usage of specified ES6 imports
```

Comment:
Sorry, my mind stopped halfway through that thought. Looks like the when no-restricted-imports derived from no-restricted-modules that *imports* replaced *modules* in the description. Both rules restrict certain modules, not the `import` or `require` itself. A thought (not necessarily a good one) would be to find a parallel grammar pattern for both descriptions so the restriction is clearly on modules and `require` or `import` fits fluently in code style.